ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/MicrosoftSentinelForm.tsx

```diff
@@ -286,21 +299,52 @@ function MicrosoftSentinelForm({
                                     onChange={onUpdateAuthMethod}
                                     isDisabled={!isEditable}
                                 />
+                                <ToggleGroupItem
```

Comment:
Layout nit: the spacing between this toggle group and the item below it is a little tight. I would suggest wrapping the items in the card body with something like `<Flex spaceItems={{ default: 'spaceItemsSm'}} direciton={{ default: 'column' }}>` for more consistent layout, or if that conflicts with the other form elements when something other than WIF is selected, a utility class for padding on the toggle group might work as well.

Reply:
I'm not sure where to put this exactly, could you please elaborate a bit more code-wise? 😅

Reply:
In this case, I think the easiest/safest fix is to add `className="pf-v5-u-pb-md"` to the ToggleGroup component directly. The should be a conflict free way to add some spacing without messing up the layout of other toggle group form elements.